include/r502_commands.h

```diff
@@ -63,6 +64,9 @@ static int32_t reg_model_pkg(uint8_t *pkg, Command command, int32_t pkg_len);
 /* Build package for Store command */
 static int32_t store_pkg(uint8_t *pkg, Command command, int32_t pkg_len);
 
+/* Build package for AuraLedConfig */
+static int32_t aura_led_config(uint8_t *pkg, Command command, int32_t pkg_len);
```

Comment:
Keep the same order of the commands as in the `cmd_func` array. This should be the last function declared.

Reply:
Done!